webui/react/src/pages/ExperimentDetails/ExperimentVisualization/LearningCurve.tsx

```diff
@@ -149,15 +149,15 @@ const LearningCurve: React.FC<Props> = ({
 
   const handlePointClick = useCallback(
     (e: MouseEvent, point: UPlotPoint) => {
-      const trialId = point ? trialIds[point.seriesIdx - 1] : undefined;
+      const trialId = point ? trialIds[point.seriesIdx] : undefined;
```

Comment:
`point` isn't optional so this can be simplified

```suggestion
      const trialId = trialIds[point.seriesIdx];
```